bigbluebutton-html5/imports/api/2.0/users/server/modifiers/userEjected.js

```diff
@@ -0,0 +1,39 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/2.0/users';
+
+export default function userEjected(meetingId, userId) {
+  check(meetingId, String);
+  check(userId, String);
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
+      ejected: true,
+      connectionStatus: 'offline',
+      listenOnly: false,
```

Comment:
This prop doesn't belong to the user object anymore. Please update the `removeUser` modifier too.